doc/usecases/elektra_web/UC_undo_redo.md

```diff
@@ -0,0 +1,20 @@
+# Use Case: Undo/Redo
+
+## Summary
+
+Title: Undo/Redo
+Scope: Configuration
+Level: User Goal
+Actors: User (usually a sysadmin)
+Brief: User undos/redos changes to a configuration.
+
+## Scenarios
+
+Precondition: Modifying keys, Adding keys, Duplicating keys, Drag & Drop keys
+Main success scenario: User successfully undos/redos a configuration change.
+Alternative scenario: Instance not online. The user is informed about the issue.
```

Comment:
Seems to be a copy&paste error from earlier versions? Instances going offline would be an error scenario?